dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -291,4 +289,40 @@ def detail_change_notification_subject(workshop)
       'Details for your upcoming Code.org workshop have changed'
     end
   end
+
+  def get_contact_text_for_teacher_follow_up(workshop)
+    regional_partner = workshop.regional_partner
+    facilitators = []
+    workshop.facilitators.each {|facilitator| facilitators << facilitator.email}
+    has_partner = regional_partner && regional_partner.contact_email
```

Comment:
You can use safe navigator `&.`. For example `regional_partner_email = workshop.regional_partner&.contact_email`